estimate-web/src/reducers/applicationContext.js

```diff
@@ -48,6 +53,11 @@ const onReceiveGroupUpdated = (state, action) => ({
     activeWorkItemId: action.activeWorkItemId
 });
 
+const onSelectedWorkItemChanged = (state, action) => ({
+    ...state,
+    selectedWorkItemId: action.selectedWorkItemId
```

Comment:
I'm not super sure about adding this to the global state. It's a component-specific state, all components that care about it should be the children of `EstimatePage`. Given that I've already refactor all of the reducers one for a cleaner state shape, I'm going to fight for this one a bit

Reply:
Excellent feedback. I've removed this from the global state.